TP n°2 (Ko y Biscaisaque)

alfo2025 opened the pull request:
TP n°2 (Ko y Biscaisaque)

uca-ed committed:
Agregados al readme

uca-ed reviewed (commented):
Por favor, revisen cómo hicieron el PR. Veo en el repo de ustedes que existen en el branch main unos archivos que pueden ser respuesta, pero deberían crear esos archivos sobre el branch del PR respetando la estructura original

alfo2025 committed:
Add functions for reading and processing adjacency matrices

Implement functions to read adjacency matrix from CSV and JSON files, and define operators for matrix manipulation.

alfo2025 committed:
Add relation analysis functions for graphs

alfo2025 committed:
Add BFS pathfinding algorithm for graphs

alfo2025 committed:
Add ArbolArreglo class with traversal methods

Implement ArbolArreglo class with methods for height and pre-order traversal.

alfo2025 committed:
Create ejercicio2

alfo2025 committed:
Create datos.json
